File: 프로그래머스/일반/21년12월/K번째수.py
def solution(array, commands):
    answer = []
    for c in commands:
        temp = sorted(array[c[0]-1:c[1]])
        print(temp)
        answer.append(temp[c[2]-1])

    return answer



a = [1, 5, 2, 6, 3, 7, 4]
b = [[2, 5, 3], [4, 4, 1], [1, 7, 3]]
print(solution(a, b))

File: 프로그래머스/일반/21년12월시작/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int x = 0;

        for (int[] c : commands){
            int[] temp = Arrays.copyOfRange(array, c[0] - 1, c[1]);
            Arrays.sort(temp);
            answer[x] = temp[c[2] - 1];
            x++;
        }

        return answer;
    }
}